module_cheatsheets/Intro_to_R_Tidyverse_cheatsheet.md

```diff
@@ -0,0 +1,82 @@
+| [`docker pull`](https://docs.docker.com/engine/reference/commandline/pull/) | Docker Pull      | Pulls an image from a docker container |
+| [`docker run`](https://docs.docker.com/v17.12/engine/reference/run/)  | Docker Run             | Runs processes in a docker container   |
+
+
+### `ggplot2`
+Read the `ggplot2` package documentation [**here**](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0). <br>
+A vignette on the usage of the `ggplot2` package can be found [**here**](https://cran.r-project.org/web/packages/ggplot2/vignettes/ggplot2-specs.html).
+
+| Library/Package                      | Piece of code                                   | What it's called       | What it does                                                                                                                                                                                      |
+|--------------------------------------|-----------------------------------------------------|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| ggplot2         | [`ggplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggplot)                   | GG Plot               | Begins a plot that is finished by adding layers.        |
+| ggplot2         | [`geom_boxplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/geom_boxplot)       | Boxplot               | Creates a boxplot when combined with ggplot()            |
+
+
+### `readr` and `tibble`
+Read the `readr` package documentation [**here**](https://www.rdocumentation.org/packages/readr/versions/1.3.0) and the package vignette [**here**](https://www.rdocumentation.org/packages/readr/versions/1.3.0/vignettes/readr.Rmd). <br>
+Read the `tibble` package documentation [**here**](https://www.rdocumentation.org/packages/tibble/versions/1.4.2) and the package vignette [**here**](https://www.rdocumentation.org/packages/tibble/versions/1.4.2/vignettes/tibble.Rmd).
+
+| Library/Package                      | Piece of code                                   | What it's called       | What it does                                                                                                                                                                                      |
+|--------------------------------------|-----------------------------------------------------|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| readr           | [`read_tsv()`](https://www.rdocumentation.org/packages/readr/versions/1.3.0/topics/read_delim)           | Read .TSV             | Reads in a .TSV file from a specified file path. This function can be tailored to read in other common types of files. I.e. read_csv(), read_rds(), etc.                                          |
```

Comment:
```suggestion
| readr           | [`read_tsv()`](https://www.rdocumentation.org/packages/readr/versions/1.3.0/topics/read_delim)           | Read TSV             | Reads in a TSV file from a specified file path. This function can be tailored to read in other common types of files, i.e. read_csv(), read_rds(), etc.                                          |
```
I would do `.tsv` when describing the extension and TSV when talking about the file.